.github/CODEOWNERS

```diff
@@ -42,8 +42,8 @@ browser/ui/views/infobars/brave_wayback_machine_* @simonhong
 components/brave_wayback_machine @simonhong
 
 # NTP sponsored images
-browser/ntp_sponsored_images @simonhong @petemill
-components/ntp_sponsored_images @simonhong @petemill
+browser/ntp_background_images @simonhong @petemill
```

Comment:
This kind of blanket ownership over an entire component is something we are specifically trying to avoid. If there are critical parts of this that need special review please list the file(s) individually

Reply:
Updated.